docs/source/user/troubleshoot.rst

```diff
@@ -874,7 +874,7 @@ breakpoint was hit, and after a ``continue`` was issued, it broke again at line
 Debugging in parallel regions
 -----------------------------
 
-The follow example is quite involved, it executes with ``gdb`` instrumentation
+The following example is quite involved, it executes with ``gdb`` instrumentation
```

Comment:
Please could this be line wrapped to 80 characters? Many thanks.

Reply:
Thanks for the comment! I have made the appropriate changes. Could you please check it out?